deployment/gke.sh

```diff
@@ -51,11 +51,16 @@ else
 fi
 
 # Set BQ dataset
-if [ "$PROJECT_ID" == "datcom-mixer-staging" ]; then
-  perl -i -pe's/BQ_DATASET/google.com:datcom-store-dev.dc_v3_staging_clustered/g' deployment.yaml
-else 
-  perl -i -pe's/BQ_DATASET/google.com:datcom-store-dev.dc_v3_clustered/g' deployment.yaml
+if [ "$PROJECT_ID" == "datcom-mixer" ]; then
+  bq_dataset_input_file="./prod_bq_dataset.txt"
+else
+  bq_dataset_input_file="./staging_bq_dataset.txt"
 fi
+while read -r line; do
+  BQ_DATASET=$line
+done < "$bq_dataset_input_file"
```

Comment:
why not?

BQ_DATASET=$(cat $bq_dataset_input_file)

Reply:
Thanks for the suggestion.

Reply:
I also updated how BT_TABLE get assigned.